src/comm/pub_handler.cpp

```diff
@@ -202,6 +206,13 @@ void PubHandler::CheckTimer() {
   }
   last_pub_time_ += std::chrono::nanoseconds(publish_interval_);
 
+  if (!time_is_steady_) {
+    if (now_time - last_pub_time_ > std::chrono::nanoseconds(publish_interval_)) {
```

Comment:
will time only jump in one direction? could maybe use `std::chrono::abs` otherwise

Reply:
on the vehicle we do one big jump forward, when chrony syncs the time for the first time. After that it is slewing the time to slowly adapt it (if there is an offset between ntp time and local time).
For testing purposes I also did a big jump backwards manually, but then the whole jetson crashed and rebooted. I think it has something to do with the real-time functions from the device, but did not find the cause yet.

Reply:
in that case i think it is fine